### PR TITLE
Self Expression Improvements

### PR DESCRIPTION
**Pull request by Geevies**
* Floating Chat messages now appear when you whisper as well.
* You can now change your floating chat message color from the IC tab.
* Fixes an extra space appearing in the textlog when whispering.
* Removed some more instances of (as ID name) that I missed before.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -958,3 +958,14 @@ default behaviour is:
 
 /mob/living/proc/needs_wheelchair()
 	return FALSE
+
+/mob/living/verb/change_floating_chat_color()
+	set name = "Change Floating Chat Color"
+	set category = "IC"
```
Comment (NonQueueingMatt) [inline code/modules/mob/living/living.dm:964]: This should be a pref rather than an IC verb

File: code/modules/mob/floating_messages.dm [hunk 2]
```diff
@@ -20,9 +20,13 @@ var/list/floating_chat_colors = list()
 	if(length(message) > limit)
 		message = "[copytext(message, 1, limit)]..."
 
-	if(!floating_chat_colors[name])
-		floating_chat_colors[name] = get_random_colour(0, 160, 230)
-	style += "color: [floating_chat_colors[name]];"
+	var/used_name = name
+	if(ismob(src))
+		var/mob/M = src
+		used_name = M.real_name
```
Comment (NonQueueingMatt) [inline code/modules/mob/floating_messages.dm:26]: nooooooo, bad, never do istype(src, ...)

**Comment by mikomyazaki** [comment]
I think this should be something defined by the viewer, not the speaker. The speaker does not need to read their own floating chat messages.

If necessary we could set the default color by their accent.

**Comment by NonQueueingMatt** [comment]
Unaddressed concerns, reopen when ready.

**Pull request closed without merging**